storage.go

```diff
@@ -556,6 +556,11 @@ func (s *Storage) writeAppend(ctx context.Context, o *Object, r io.Reader, size
 }
 
 func (s *Storage) writeMultipart(ctx context.Context, o *Object, r io.Reader, size int64, index int, opt pairStorageWriteMultipart) (n int64, part *Part, err error) {
+	if index < multipartNumberMinimum || index > o.MustGetMultipartNumberMaximum() {
```

Comment:
only find `MustGetMultipartNumberMaximum`, there is no function like `MustGetMultipartNumberMinimum`

Reply:
Because minimum is the same for services: `0`, so we don't need to add this meta in the object.

`MustGetMultipartNumberMaximum` and `GetMultipartNumberMaximum` should be used by users. As service provider, we don't need to get them from Object, we can use the const value we defined.